src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -160,13 +186,17 @@ private static FrameworkInstallation GetFromV4(RegistryKey subKey, string subKey
                 },
                 Version = version,
                 ServicePack = subKey.GetInt("SP"),
-                Release = hasRelease ? release : null as int?
+                Release = hasRelease ? release : null
             };
         }
 
         // https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#to-find-net-framework-versions-by-querying-the-registry-in-code-net-framework-45-and-later
-        internal static int? Get45PlusLatestInstallationFromRegistry()
+        internal static int? Get45PlusLatestInstallationFromRegistry(bool registryLock)
         {
+            if (registryLock)
+            {
+                return null;
+            }
```

Comment:
What's the point of passing a parameter that just makes the function exit immediately? Wouldn't it be better to _just not_ call the function instead?